.github/pull_request_template.md

```diff
@@ -12,20 +10,28 @@
 
 ### Addressed issue
 
-<!-- Link to the issue, if present. E.g 
+<!-- Link to the issue, if present. E.g. 
     Closes #XYZ
 -->
 
-### Checklist
+### [MANDATORY] PR Checklist
+
+<!-- IMPORTANT: Before submitting the PR, please take a look at the CONTRIBUTING.md file for further details regarding standards and conventions adopted in this repository. You can find it here: https://github.com/mia-platform/design-system/blob/main/CONTRIBUTING.md -->
```

Comment:
we could place it as the first checklist item?

Reply:
I made an update, see if it works better :)

Reply:
yep looks good to me!